presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionSuite.scala

```diff
@@ -1193,7 +1193,7 @@ class CompletionSuite extends BaseCompletionSuite:
          |  val x = Bar[M](new Foo[Int]{})
          |  x.bar.m@@
          |""".stripMargin,
-      """|map[B](f: A => B): Foo[B]
+      """|map[B](f: Int => B): Foo[B]
```

Comment:
Why is the test changed here?

Reply:
Because we've fixed the prefix, using `x.bar : Foo[Int]` instead of `Foo[A]`.